File: C++/A1/023.cpp
#include <iostream>
using namespace std;
int main(){
    int A;
    char B;
    cin >> A >> B;
    
    if(toupper(B) == 'C'){
        if(A <= 0)cout << "solid";
        else if(A >= 100)cout << "gas";
        else cout << "liquid";
    }else if(toupper(B) == 'F')
        if(A <= 32)cout << "solid";
        else if(A >= 212)cout << "gas";
        else cout << "liquid";

}   

File: Python/A1/023.py
A,B = int(input()),input().lower()
if(B == 'c'):
    if(A<=0):print("solid")
    elif(A>=100):print("gas")
    else:print("liquid")
elif(B == 'f'):
    if(A<=32):print("solid")
    elif(A>=212):print("gas")
    else:print("liquid")